python/demo/gmsh/demo_msh.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2020 Garth N. Wells and Jørgen S. Dokken
```

Comment:
This file doesn't look ready.

Reply:
I can add some documentation for each of the mesh generation function explaining what kind of meshes they create.